test/KoreBuild.FunctionalTests/SimpleRepoTests.cs

```diff
@@ -52,14 +52,9 @@ public void FullBuildCompletes()
             {
                 Assert.Empty(reader.GetFiles().Where(p => Path.GetExtension(p).Equals(".pdb", StringComparison.OrdinalIgnoreCase)));
             }
-
-            // /t:TestNuGetPush
-            Assert.True(File.Exists(Path.Combine(app.WorkingDirectory, "obj", "tmp-nuget", "Simple.CliTool.1.0.0-beta-0001.nupkg")), "Build done a test push of all the packages");
-            Assert.True(File.Exists(Path.Combine(app.WorkingDirectory, "obj", "tmp-nuget", "Simple.Lib.1.0.0-beta-0001.nupkg")), "Build done a test push of all the packages");
-            Assert.True(File.Exists(Path.Combine(app.WorkingDirectory, "obj", "tmp-nuget", "Simple.Sources.1.0.0-beta-0001.nupkg")), "Build done a test push of all the packages");
         }
 
-        [Fact]
+        [Fact(Skip = "Blocked on https://github.com/dotnet/sdk/issues/2867")]
```

Comment:
This issue is closed. Is it necessary to skip this test?

Reply:
Issue is closed, but the fix won't be in an SDK until 3.0 Preview 3.